### ISSUE TITLE
keep your variables in camel case instead of underscores

### ISSUE THREAD

**Issue by rampallydheeraj** [srikarrampa/RestaurantSearchEngine#1]
https://github.com/srikarrampa/RestaurantSearchEngine/blob/486741e6bd2a791bb401be08a8aa25b6add76df2/src/main/java/com/springboot/type/Type.java#L22

**Comment by srikarrampa** [comment]
variables wouldn't match with the databases columns. So left it in the same manner

**Comment by rampallydheeraj** [comment]
There is an @ Column annotation which you should be using

**Comment by srikarrampa** [comment]
Yes, I made changes to the structure and added @ Column and @ Transactional
